lua/lazyvim/plugins/extras/lang/zig.lua

```diff
@@ -0,0 +1,66 @@
+return {
+  {
+    "NTBBloodbath/zig-tools.nvim",
+    ft = "zig",
+    opts = {},
+    dependencies = {
+      {
+        "akinsho/toggleterm.nvim",
```

Comment:
I'm not a fan of pulling in toggleterm just for zig. It also is not setup, so will cause issues

Reply:
If i remove toggleterm zig-tools give me this error every time (no config needed to prevent this error) :

```
   Error  17:50:10 notify.error lazy.nvim Failed to run `config` for zig-tools.nvim

...e/LazyVim/lazy/zig-tools.nvim/lua/zig-tools/commands.lua:14: module 'toggleterm.terminal' not found:
	no field package.preload['toggleterm.terminal']
cache_loader: module toggleterm.terminal not found
cache_loader_lib: module toggleterm.terminal not found
	no file './toggleterm/terminal.lua'
	no file '/usr/share/luajit-2.1/toggleterm/terminal.lua'
	no file '/usr/local/share/lua/5.1/toggleterm/terminal.lua'
	no file '/usr/local/share/lua/5.1/toggleterm/terminal/init.lua'
	no file '/usr/share/lua/5.1/toggleterm/terminal.lua'
	no file '/usr/share/lua/5.1/toggleterm/terminal/init.lua'
	no file './toggleterm/terminal.so'
	no file '/usr/local/lib/lua/5.1/toggleterm/terminal.so'
	no file '/usr/lib/lua/5.1/toggleterm/terminal.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file './toggleterm.so'
	no file '/usr/local/lib/lua/5.1/toggleterm.so'
	no file '/usr/lib/lua/5.1/toggleterm.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'

# stacktrace:
  - /zig-tools.nvim/lua/zig-tools/commands.lua:14
  - /zig-tools.nvim/lua/zig-tools/autocmds.lua:8
  - /zig-tools.nvim/lua/zig-tools.lua:9
  - /usr/share/nvim/runtime/filetype.lua:31
  - /usr/share/nvim/runtime/filetype.lua:30
  - vim/_editor.lua:0
  - /telescope.nvim/lua/telescope/actions/set.lua:200 _in_ **run_replace_or_original**
  - /telescope.nvim/lua/telescope/actions/mt.lua:65 _in_ **run_replace_or_original**
  - /telescope.nvim/lua/telescope/actions/mt.lua:65 _in_ **run_replace_or_original**
  - /telescope.nvim/lua/telescope/actions/mt.lua:65 _in_ **key_func**
  - /telescope.nvim/lua/telescope/mappings.lua:257

```

Reply:
From another similar pr, zig-tools.vim could also be used as it's also by the authors of zig themselves and doesn't have toggleterm as a hard dependency. But, as you may guess, it's written in Vimscript, not in Lua